packages/a11y-testing/src/definitions/Link/linkBehaviorDefinition.ts

```diff
@@ -13,10 +13,15 @@ export const linkBehaviorDefinition: Rule[] = [
     .hasAttribute('type', 'button')
     .description(`if element does not have href and is rendered as a 'button'.`),
   BehaviorRule.root()
-    .forProps({ as: 'div' })
+    .forProps({ as: 'button', href: '#' })
     .hasAttribute('role', 'link')
+    .hasAttribute('type', 'button')
+    .description(`if element is forced to render as a 'button' even if it has an href.`),
+  BehaviorRule.root()
+    .forProps({ as: 'a' })
+    .doesNotHaveAttribute('role')
     .hasAttribute('tabindex', '0')
```

Comment:
This seems wrong -- if it's rendered as an `a` it shouldn't have a `tabindex` attribute. Actually, scratch that -- it also shouldn't have an explicit `tabindex` (unless passed in through props) whether it's a link or a button.

This particular test (a, no href, no role) is just a good example of where it creates a problem -- `<a tabindex="0">` is the same as `<span tabindex="0">`, and is an a11y fail.

Reply:
Yup, a separate a11y review is needed for `Link`, but let's not block this PR because of that 😄